notebooks/book1/04/samplingDistributionGaussianShrinkage.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    latexify(width_scale_factor=2, fig_height=1.85, font_size=7)
</pre><p>Don't decrease font_size of all labels, decrease only those labels which are overflowing, contact anand how to do this, I explained him once</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>